ipyleaflet/leaflet.py

```diff
@@ -392,6 +392,25 @@ class Path(VectorLayer):
     opacity = Float(1.0, min=0.0, max=1.0).tag(sync=True, o=True)
 
 
+class AntPath(VectorLayer):
+    _view_name = Unicode('LeafletAntPathView').tag(sync=True)
+    _model_name = Unicode('LeafletAntPathModel').tag(sync=True)
+
+    locations = List().tag(sync=True)
+
+    # Options
+    use = Unicode('polyline').tag(sync=True, o=True)
```

Comment:
Maybe use an `Enum` so you can validate the value. `Enum(values=['polyline', 'polygon', 'circle'], default_value='polyline')`

Reply:
Good point, thanks.